components/components/navigation/menu-main/menu-main.css

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file
+ * This file is used to style the main menu.
+ */
+
+/*
+  Main Menu
+
+  This section presents the Main Menu of the website.
+
+  Markup: menu-main.html.twig
+
+  Style guide: components.navigation.main_menu
+*/
+
+.menu-main__toggle {
+  width: 40px;
+  height: 40px;
+  line-height: 1;
+  padding: 0;
+  border: none;
+  text-align: left;
+  background-color: transparent;
+}
+.menu-main {
+  margin: 0;
+  padding: 0;
+  max-height: 0;
+  overflow: hidden;
+  list-style-type: none;
+  font-family: 'Scope One';
+  font-weight: normal;
+  font-size: 1.266rem;
+  line-height: 1.2;
+  color: #000;
+  transition: max-height 0.5s ease-in;
+}
+.menu-main.is-active {
+  /**
+   * An exact max-height value is must be set to make possible to animate the menu display.
+   * It my not be much larger than the content because that would break the animation.
+   * See: https://css-tricks.com/using-css-transitions-auto-dimensions/
+   */
+  max-height: 18.75rem;
+  overflow-y: auto;
+}
+.menu-main__item {
+  margin-top: 0.8em;
+}
+.menu-main__link {
+  display: block;
+  padding-bottom: 0.15em;
+  text-decoration: none;
+  border-bottom: solid 0.15em transparent;
+  color: inherit;
+  transition: all 0.2s;
```

Comment:
Would be great to get feedback about this from @kjays since this wasn't something that I was seeing in the original design

Reply:
OK. Can we get him to check? :)

Reply:
@lauriii Hm, are you talking abou the design of the links? It was in the sketch and in the pdf styleguide too. 

And he provided a "mobile" design on Slack some days ago 
![umami - front - mobile - nav open](https://user-images.githubusercontent.com/1244043/31885760-f410d10a-b7f1-11e7-91fe-3e68f27a9fe4.jpg)
so I updated the code according to it too.